themes/default/content/learn/building-with-pulumi/stack-outputs/index.md

```diff
@@ -42,7 +42,7 @@ export const url = pulumi.interpolate`http://localhost:${frontendPort}`;
 {{% choosable language python %}}
 
 ```python
-pulumi.export("url", pulumi.format("http://localhost:{0}", frontend_port))
+pulumi.export("url", f"http://localhost:{frontend_port}")
```

Comment:
```suggestion
pulumi.export("url", Output.format("http://localhost:{0}", frontend_port))
```

frontend_port would generally be an Output value and so can't be used with f-strings. There is a `format` method for this but it's on the `Output` class not the root pulumi module.

Reply:
Hm, I thought `frontend_port` is a simple integer in this case:

```python
frontend_port = config.require_int("frontend_port")
```

So, should I update the PR to include full method path, like `pulumi.Output.format`?
Because `Output` class is not imported in that context.

Reply:
Yeh in this case it would be fine to f-string it, but keeping the style of the the Typescript example on this page I think these examples are intended to be more general. 

And yes if Output isn't imported directly anywhere than `pulumi.Output.format` would be more correct.